File: C++/Decimal to binary.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
  int num;
  cout << "Enter number: ";
  cin >> num;

  int i = 0;
  int answer = 0;

  while (num != 0)
  {
    int bit = num & 1;

    answer = (bit * pow(10, i)) + answer;

    num = num >> 1;
    i++;
  }

  cout << "Answer: " << answer << endl;

  return 0;
}

File: JavaScript/decimalToBinary.js
const prompt = require("prompt-sync")();

let num = Number(prompt("Enter num: "));

let ans = 0;
let i = 0;

while (num > 0) {
  let bit = num & 1;
  ans = bit * Math.pow(10, i) + ans;
  num = num >> 1;
  i++;
}

console.log(`Answer is: ${ans}`);
